### ISSUE TITLE
Update info hanging RC1 -> RC2

### ISSUE THREAD

**Issue by jeroenmaelbrancke** [openvstorage/framework#1997]
# Problem
I'm testing the update from bighorn-rc-1 to rc-2.
After refreshing the update page the loading information did not disappear.  
![image](https://user-images.githubusercontent.com/2900612/37345458-4ee24628-26cd-11e8-8916-12ff3b5be068.png)

```
jquery-3.2.1.js:3860 jQuery.Deferred exception: Cannot read property 'length' of undefined TypeError: Cannot read property 'length' of undefined
    at Object.za (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:14:286)
    at a.a.Bb (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:119:207)
    at Object.w (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:39:392)
    at a.B.i (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:114:5)
    at Function.Pc (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:51:355)
    at Function.Qc (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:51:133)
    at Function.aa (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:50:482)
    at Function.ac (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:50:335)
    at Function.notifySubscribers (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:37:108)
    at Function.fa (https://10.100.50.18/lib/knockout/knockout-3.4.0.js:40:499) undefined
```

Since openvstorage version 2.11.1 the method get_update_information_core is deprecated and is no longer available on the UpdateController but the api call get_update_status still use this method. (see info)

https://github.com/openvstorage/framework/blob/2.11.1/webapps/api/backend/views/storagerouters.py#L369

# Info
```
In [32]: for id in client.get(api='/storagerouters')['data']:
    print client.get(api='/storagerouters/{0}/get_update_status/'.format(id))
   ....:     
---------------------------------------------------------------------------
HttpException                             Traceback (most recent call last)
<ipython-input-32-c4126d794386> in <module>()
      1 for id in client.get(api='/storagerouters')['data']:
----> 2     print client.get(api='/storagerouters/{0}/get_update_status/'.format(id))
      3 

/usr/lib/python2.7/dist-packages/ovs_extensions/api/client.pyc in get(self, api, params)
    186         :type params: dict
    187         """
--> 188         return self._call(api=api, params=params, fct=requests.get)
    189 
    190     def post(self, api, data=None, params=None):

/usr/lib/python2.7/dist-packages/ovs_extensions/api/client.pyc in _call(self, api, params, fct, **kwargs)
    165         headers, url = self._prepare(params=params)
    166         try:
--> 167             return self._process(fct(url=url.format(api), headers=headers, verify=self._verify, **kwargs))
    168         except HttpForbiddenException:
    169             if self._volatile_client is not None:

/usr/lib/python2.7/dist-packages/ovs_extensions/api/client.pyc in _process(self, response, overrule_raw)
    153                 raise HttpNotFoundException(message, '')
    154             else:
--> 155                 raise HttpException(status_code, message)
    156 
    157     def _call(self, api, params, fct, **kwargs):

HttpException: Status code: 500 - Error: type object 'UpdateController' has no attribute 'get_update_information_core' (internal_server)'.
```

# Versions
```
ii  openvstorage                         2.11.1-1                           amd64        OpenvStorage
ii  openvstorage-backend                 1.11.0-1                           amd64        OpenvStorage Backend plugin
ii  openvstorage-backend-core            1.11.0-1                           amd64        OpenvStorage Backend plugin core
ii  openvstorage-backend-webapps         1.11.0-1                           amd64        OpenvStorage Backend plugin Web Applications
ii  openvstorage-core                    2.11.1-1                           amd64        OpenvStorage core
ii  openvstorage-extensions              0.3.0-1                            amd64        Extensions for Open vStorage
ii  openvstorage-health-check            3.6.5-1                            amd64        Open vStorage HealthCheck
ii  openvstorage-webapps                 2.11.1-1                           amd64        OpenvStorage Web Applications
```

**Comment by sjovdnbos** [comment]
The stacktrace provided by Jeroen is irrelevant. He used a deprecated api call ``` get_update_status ```, but this call is nowhere used in the framework 2.11 version. 
What is messing up the update info page is failure of loading of the storagenodes info, this is all i know for now.

**Comment by sjovdnbos** [comment]
The problem lies not in the update logic but in the gui (update page) hanging, but I am for the moment not able to solve the issue. 

## Workaround:
``` UpdateController.merge_package_information() ```
Should return a dict like 
``` 
ip: {component1 :{...},
     component2: {...},
     component3: {...},
ip2: 
          ...}
```
Use these components and run on a master node:
``` ovs update component1,component2,...```
This should update all nodes.

**Comment by JeffreyDevloo** [comment]
Fixed by #2003 
Released in https://github.com/openvstorage/framework/releases/tag/2.11.5
